From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: David Sanders <dsanders11@ucsbalum.com>
Date: Wed, 30 Oct 2024 19:14:19 -0700
Subject: fix: don't use forward declaration header

Fails to build from source unless the full header is included. Being upstreamed at https://chromium-review.googlesource.com/c/chromium/src/+/5981701.

Refs: https://chromium-review.googlesource.com/c/chromium/src/+/5868889

diff --git a/content/browser/shared_storage/shared_storage_saved_query_data.h b/content/browser/shared_storage/shared_storage_saved_query_data.h
index 206c720fcb797eb36e13183044e13d4e2159b48f..3f37b06492305e279eb46886ddfa6f9a170eaf6e 100644
--- a/content/browser/shared_storage/shared_storage_saved_query_data.h
+++ b/content/browser/shared_storage/shared_storage_saved_query_data.h
@@ -9,7 +9,7 @@
 
 #include <queue>
 
-#include "base/functional/callback_forward.h"
+#include "base/functional/callback.h"
 #include "content/common/content_export.h"
 
 namespace content {
